BraintreeCore/src/test/java/com/braintreepayments/api/ConfigurationLoaderUnitTest.kt

```diff
@@ -0,0 +1,171 @@
+package com.braintreepayments.api
+
+import android.util.Base64
+import io.mockk.*
+import org.robolectric.RobolectricTestRunner
+import org.json.JSONException
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.lang.Exception
+
+@RunWith(RobolectricTestRunner::class)
+class ConfigurationLoaderUnitTest {
+    private var configurationCache: ConfigurationCache = mockk(relaxed = true)
+    private var braintreeHttpClient: BraintreeHttpClient = mockk(relaxed = true)
+    private var callback: ConfigurationLoaderCallback = mockk(relaxed = true)
+    private var authorization: Authorization = mockk(relaxed = true)
+
+    @Test
+    fun loadConfiguration_loadsConfigurationForTheCurrentEnvironment() {
+
+        every{ authorization.configUrl } returns "https://example.com/config"
+
+        val sut = ConfigurationLoader(braintreeHttpClient, configurationCache)
+        sut.loadConfiguration(authorization, callback)
+
+        val expectedConfigUrl = "https://example.com/config?configVersion=3"
+        val callbackSlot = slot<HttpResponseCallback>()
+        verify {
+            braintreeHttpClient[
+                    expectedConfigUrl,
+                    null,
+                    authorization,
+                    HttpClient.RETRY_MAX_3_TIMES,
+                    capture(callbackSlot)
+            ]
```

Comment:
```suggestion
            braintreeHttpClient.get(
                    expectedConfigUrl,
                    null,
                    authorization,
                    HttpClient.RETRY_MAX_3_TIMES,
                    capture(callbackSlot)
            )
```
The Kotlin conversion tools refactors these to use brackets for some reason.